pages/dashboard/mentor.page.js

```diff
@@ -50,14 +53,128 @@ function MentorDashboard() {
     }
   }, [window.location.search, history]);
 
+  //guided tour
+
+  const { setIsOpen, setSteps, setCurrentStep } = useTour();
+
+  useEffect(() => {
+    setCurrentStep(0);
+    const mentorTour = localStorage.getItem("mentorTour");
+    if (mentorTour) return;
+    setIsOpen(true);
+
+    setSteps([
+      {
+        selector: "#mentorHome",
+        content: (
+          <div className="tw-flex tw-flex-col tw-items-center tw-justify-center">
+            <div className="tw-flex tw-justify-center tw-items-center">
```

Comment:
can you please move this to dashboard components as tour component!? and it should be shows only once to user/mentor
you can use local storage for this!

Reply:
alright